src/pkg/packaging/windows/sharedframework/bundle.wxs

```diff
@@ -27,7 +27,7 @@
 
     <swid:Tag Regid="microsoft.com" InstallPath="[DOTNETHOME]" />
 
-    <Variable Name="DOTNETHOME" Type="string" Value="[ProgramFiles6432Folder]dotnet" bal:Overridable="no" />
+    <Variable Name="DOTNETHOME" Type="string" Value="[$(var.Program_Files)]dotnet" bal:Overridable="no" />
```

Comment:
There is no change in the Install location for Bundles across Windows architecture.

As of today, X86 bundle on x64 Windows has DOTNETHOME value set to Program Files , However, during MSI installation, Windows Installer sets the right value to Program Files (x86) . 

MSI (s) (24:D8) [10:02:27:751]: WIN64DUALFOLDERS: 'C:\Program Files (x86)\' will substitute 17 characters in 'C:\Program Files\dotnet\' folder path. (mask argument = 0, the folder pair's iSwapAttrib member = 0).
MSI (s) (24:D8) [10:02:27:751]: PROPERTY CHANGE: Modifying DOTNETHOME property. Its current value is 'C:\Program Files\dotnet'. Its new value: 'C:\Program Files (x86)\dotnet\'.

We are ensuring that DOTNETHOME variable has correct value set to reflect the correct install location. This value is now used to show the install location of a bundle in Installer UI. 

Program_Files is a variable that is set correctly depending on the Package arch https://github.com/dotnet/core-setup/blob/master/src/pkg/packaging/windows/sharedframework/variables.wxi#L20

On Windows x64 :  
  - x86 runtime installer gets installed on Program Files (x86) 
  - x64 runtime installer gets installed on Program Files

On Windows x86 :
- x86 runtime installer gets installed on Program Files